src/main/java/frc/robot/subsystems/IntakeMechanisum.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.REVPhysicsSim;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class IntakeMechanisum extends SubsystemBase {
+  CANSparkMax intake;
+  
+  /** Creates a new IntakeMechanisum. */
+  public IntakeMechanisum() {
+
+    intake = new CANSparkMax(Constants.CANSparkMaxID.intake, MotorType.kBrushless);
+
+    float stallTorque = Constants.Misc.stallTorque;
+    float freeSpeed = Constants.Misc.freeSpeed;
+
+    REVPhysicsSim.getInstance().addSparkMax(intake, stallTorque, freeSpeed);
```

Comment:
instead of this version of `addSparkMax`, you can also use `addSparkMax(motor, DCMotor.getNEO(1));`, and it will populate all of the NEO parameters for you. `DCMotor` also has methods for other common motors.